### ISSUE TITLE
InstallWindowsAgent.ps1 - vcredist installers not executing during installation.

### ISSUE THREAD

**Issue by samlearnscode** [TheJumpCloud/support#219]
Preface: Forgive me if I have this wrong as I am very new to Powershell and Jumpcloud. 

We are having trouble running this script silently, as it fails to install the 2013 vc redist dependencies.

```
$msvc2013x64File = 'vc_redist.x64.exe'
$msvc2013x86File = 'vc_redist.x86.exe'
```

These lines do not point to the same file names we get from the Microsoft links.

```
$msvc2013x64File = 'vcredist_x64.exe'
$msvc2013x86File = 'vcredist_x86.exe'
```

**Comment by btstevens** [comment]
Hello samlearnscode, 

How are you running the InstallWindowsAgent.ps1 script and at what step are you receiving an error?

We suggest you save the InstallWIndowsAgent.ps1 to a location e.g. C:\ and then call it passing in the JumpCloud Connect key found on the console.

`C:\InstallWIndowsAgent.ps1 -JumpCloudConnectKey 89736b21ad5829h90p128f35der`

It should then return something like below:
![image](https://user-images.githubusercontent.com/35233498/78373943-78264f00-7588-11ea-956b-bfab7113bcae.png)

If you are getting errors regarding the vc redist, make sure the C:\Windows\Temp\ folder is cleared out and also, try and uninstall any  vc redist versions in add remove programs first. Please let me know what version that was already on the system as it might be conflicting and we can try and resolve that.

**Comment by davidhartman-giantoak** [comment]
I recently encountered an issue similar to what is noted above.  In my case, I run the script as noted above by bstevens and it says the 2 prereq installs completed and then attempts to install the JC agent and comes back with JumpCloud Agent Installer Failed.  
After this occurred a window came up asking if I wanted to install the Visual C++ prereqs, which was odd as the output suggested that it was already done.  I went ahead and let Windows do the install.  Oddly, after this I checked the JC agent install log and it appears the install did in fact complete with no issues and the device showed up in the console.  So I suspect there is something with the check that happens after the installs complete.
